rust/src/action/mod.rs

```diff
@@ -149,6 +149,22 @@ pub struct StatsParsed {
     pub null_count: HashMap<String, DeltaDataTypeLong>,
 }
 
+/// Delta AddCDCFile action that describes a parquet CDC data file.
+#[derive(Serialize, Deserialize, Clone, Debug, Default)]
+#[serde(rename_all = "camelCase")]
+pub struct AddCDCFile {
+    /// A relative path, from the root of the table, to a CDC file
```

Comment:
Technically, it's allowed to be absolute:

```suggestion
    /// A relative path, from the root of the table, or an
    /// absolute path to a CDC file
```